app/services/AuthService.ts

```diff
@@ -13,5 +17,27 @@ export const AuthService = {
 			return true;
 		}
 		return false;
+	},
+
+    async validateLogin(uid: string, password) {
+
+		
+	    var data
+		await axios.post('hslinux:38383/api/v1/auth', {
+		uid: uid,
+		password: password
+		  })
+		  .then(function (response) {
+		data = console.log(response.data);
+			
+		  })
+		  .catch(function (error) {
+			if (error.response) {
+				console.log(error.response.status);
+			  }
```

Comment:
Good looking out for catching the errors, but we should convert this into our own exception and throw that. I think you already defined some exceptions or have ones ready to go for this, so just look at those.